compiler/z/codegen/S390GenerateInstructions.cpp

```diff
@@ -3380,3 +3240,74 @@ TR::Instruction *generateZeroVector(TR::Node *node, TR::CodeGenerator *cg, TR::R
    {
    return generateVRIaInstruction(cg, TR::InstOpCode::VREPI, node, vecZeroReg, 0, 3);
    }
+
+#ifdef J9_PROJECT_SPECIFIC
+
+/**
+ * \brief
+ *    Determines if an instruction can throw a decimal overflow exceptions.
+ *
+ * \param op
+ *    The instruction to check.
+ *
+ * \return
+ *    true if \p op is designed to throw decimal overflow exception according to the PoP; false otherwise.
+ *
```

Comment:
OK, fixed. 
I realized that it's a bit confusing to name it like such. Given that the TR and OMR naming convention doesn't specify method naming, I'm therefore following google's naming convention, which is something that I'm familiar with: method names should start with verbs; and for methods that return booleans, use 'is' in front of them.